### ISSUE TITLE
READ THIS!

### ISSUE THREAD

**Issue by charlymr** [charlymr/XlsxReaderWriter#4]
### If you are using this project, please consider offering some of your time to contribute to it:
- Update the ReadME file
- Create a Wiki page with how to use
- Add documentation to the header file
- Check the TODO list

**Comment by ratranqu** [comment]
Found this which may be of help: https://github.com/joelparkerhenderson/demo_swift_excel_xlsx_reader_writer

**Comment by maureeungaro** [comment]
charlymr are you actively working on this, your fork seems more up-to-date than master. Keeping this up-to-date with swift / Xcode would be ideal.

**Comment by charlymr** [comment]
@maureeungaro Yes I am actively trying to keep it running as I am using it myself. It is fully compatible with swift and come as a framework. 

I will needs to add the null nonull flags for even better support, but I am missing time and open source doesn’t pray the bill!

Please if you want to contribute I will be more than happy to merge any pull request

**Comment by charlymr** [comment]
FYI try to use the beta branch it is up to date with the lates Xcode

**Comment by maureeungaro** [comment]
Yes I will use 2.4.beta4 - or keep current with your latest. With master there were a couple of problems. I will start by reporting them if they're still present here. With some guidance I may be able to fix those as well.

**Comment by maureeungaro** [comment]
Latest branch not updated, hopefully you're still working on it?

**Comment by charlymr** [comment]
Hello @maureeungaro, yes I am planning updating it. Just been busy with delivery of another project. Thanks for your patience
